fairroot/basemq/devices/FairMQUnpacker.h

```diff
@@ -88,7 +88,7 @@ class FairMQUnpacker : public fair::mq::Device
         }
 
         // check if given channel exist
```

Comment:
Please also update the copyright header?

Reply:
done